benchmark_runner/benchmark_operator/benchmark_operator_workloads.py

```diff
@@ -228,26 +231,28 @@ def tear_down_vm_after_error(self, yaml: str, vm_name: str):
     def system_metrics_collector(self, workload: str):
         """
         This method run system metrics collector
-        @param workload:
+        @param workload: the workload
         :return:
         """
+        if self.__run_type == 'test_ci':
+            es_index = 'system-metrics-test-ci'
+        else:
+            es_index = 'system-metrics'
```

Comment:
Why is this pulled out of the `if self.es_host`?  This isn't needed if we're not uploading data.